Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/AltitudeReferenceSystem.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT
+{
+    /// <summary>
+    /// <see cref="Windows.Devices.Geolocation.AltitudeReferenceSystem"/>
+    /// </summary>
+    public enum AltitudeReferenceSystem
```

Comment:
@mcleanbyron Here are more WinRT types that are shadowed. Is the present documentation sufficient?